src/components/Title/Title.js

```diff
@@ -87,6 +91,13 @@ const Title = (props: Props) => {
 
   const noBlueDotNeeded = noBlueDot || !title;
 
+  /**
+   *  `fontFamily` has to stay here as it affects font rendering
+   *  otherwise if it's taken then once font is being shorten with ellipsis
+   *  on Android it gets cut
+   */
+  const titleStylesFix = { ...titleStyles, fontFamily: `Aktiv Grotesk App${Platform.OS === 'android' ? '_bold' : ''}` };
```

Comment:
I would suggest to create an styled component from Text and set this styles there. Also `titleStylesFix` could be renamed to, for example, `AktivText`

```
const AktivText = styled(Text)`
  fontFamily: 'Aktiv Grotesk' App${Platform.OS === 'android' ? '_bold' : ''};
`
```

Otherwise this looks good.

Reply:
👍